MapboxGeocoder/MBGeocoder.swift

```diff
@@ -268,7 +268,10 @@ open class Geocoder: NSObject {
         request.setValue(userAgent, forHTTPHeaderField: "User-Agent")
         return URLSession.shared.dataTask(with: request) { (data, response, error) in
 
-            guard let data = data else { return }
+            guard let data = data else { 
+                errorHandler(error)
```

Comment:
Great catch. Would you mind adding a test for this case? Also, you have to return the `error` as? NSError

Reply:
got it, i will try this later today.

Reply:
I see https://github.com/mapbox/MapboxGeocoder.swift/pull/175/commits/51300ab02f1b5918ac19a146bbe4a94acad19ad5 refers to a test case, but I don't see one. Is this still something you could add?

Thanks for the PR, this is important missing functionality.